definitions/variable/emissions/emissions.yaml

```
@@ -3,24 +3,27 @@
 # Gross emissions are only defined for CO2.
 
 - Emissions|{Level-1 Species}:
-    description: Emissions of {Level-1 Species}
+    description: Emissions of {Level-1 Species}, net of emissions to the atmosphere and the removal of {Level-1 Species} from the atmosphere.
     unit: "{Level-1 Species}"
     components:
       - Emissions|{Level-1 Species}|AFOLU
       - Emissions|{Level-1 Species}|Energy
       - Emissions|{Level-1 Species}|Industrial Processes
       - Emissions|{Level-1 Species}|Waste
-      - Emissions|{Level-1 Species}|Capture and Removal
+      - Emissions|{Level-1 Species}|Other Capture and Removal
+      - Emissions|{Level-1 Species}|Product Use
       - Emissions|{Level-1 Species}|Other
 - Gross Emissions|CO2:
-    description: Gross emissions of carbon dioxide (CO2), not accounting for negative
-      emissions from bioenergy with CCS (BECCS) or agriculture, forestry
-      and other land use (AFOLU)
+    description: Gross emissions of carbon dioxide (CO2), not accounting for 
```

Why remove the clear reference to BECCS and negative emissions? Also it's not clear what "not accounting" means in this context...

---

> Why remove the clear reference to BECCS and negative emissions?

I think it is because there are more negative emissions options than just BECCS now. Rather than repeating them all, it's quicker and clearer to just say, "excluding all negative".

---

Ok. Given that negative emissions is not just BECCS, I do think it needs to be somehow clear that all removals are excluded? Something like the below?

```suggestion
    description: Gross emissions of carbon dioxide (CO2). This is emissions only. All removals (for example, removals from BECCS or direct air capture (DAC)) are not included here.
```

---

In 4ab62596c089f3981e2413ada6b137d86eea18a9